### ISSUE TITLE
eye is no longer supported in new julia

### ISSUE THREAD

**Issue by kuro26** [JuliaOpt/Convex.jl#331]
examples-->basic_usage， first linear programming, eye() function still used here.

**Comment by ericphanson** [comment]
Yes, unfortunately the examples are out of date. Pull requests to update them are very welcome!

**Comment by ericphanson** [comment]
By the way, the examples have been updated and are now available from the documentation page: https://www.juliaopt.org/Convex.jl/stable/ (see down on the sidebar).
